docs/content/usage.md

```diff
@@ -182,6 +182,37 @@ $ gomplate -c .=http://xkcd.com/info.0.json -i '<a href="{{ .img }}">{{ .title }
 <a href="https://imgs.xkcd.com/comics/diploma_legal_notes.png">Diploma Legal Notes</a>
 ```
 
+### `--missing-key`
+
+Changes behavior if key not exist in the datasource.
```

Comment:
this isn't just for datasources, so should be written to be more general, and I think a reference to https://pkg.go.dev/text/template#Template.Option is worthwhile